### DISCUSSION TITLE
How to create a test-route for mail/notifications testing

### DISCUSSION THREAD

**Discussion by el-schneider** [duncanmcclean/simple-commerce#765] [Help] [answered]
I'd like to test custom email templates. I've followed the guide here https://simple-commerce.duncanmcclean.com/notifications and managed to publish all the blade templates I need through: `php artisan vendor:publish --tag=laravel-mail`

Could someone (@duncanmcclean 😀) give me an example of a test route I could use to send mail for testing easily?

Something like this (only that this doesn't work)

```php
// set up a test-email route for sending notification email
Route::get('/test-email', function () {
    $order = \DoubleThreeDigital\SimpleCommerce\Facades\Order::find('1b22a351-ffb2-428c-be05-715856560b8e');

    $order->notify(new App\Notifications\OrderPaidNotification($order));
});

```

**Comment by duncanmcclean** [comment, marked as answer]
When you say it's not working, what's the issue you're running into?

**Reply by el-schneider** [reply]
I get `Call to undefined method DoubleThreeDigital\SimpleCommerce\Orders\Order::notify()`

**Reply by duncanmcclean** [reply]
I'm on my phone today but maybe try.. $order->customer()->notify(...)

**Reply by el-schneider** [reply]
Nice! That did the trick ...
